pkg/console/starter/starter.go

```diff
@@ -99,7 +104,7 @@ func RunOperator(clientConfig *rest.Config, stopCh <-chan struct{}) error {
 	oauthInformers := oauthinformers.NewSharedInformerFactoryWithOptions(
 		oauthClient,
 		resync,
-		oauthinformers.WithTweakListOptions(tweakListOptions),
+		oauthinformers.WithTweakListOptions(tweakOAuthListOptions),
```

Comment:
@enj so this is curious.  my RBAC is 
```
kind: ClusterRole
apiVersion: rbac.authorization.k8s.io/v1
metadata:
  name: console-operator
rules:
- apiGroups:
  - oauth.openshift.io]
  resources:
  - oauthclients
  verbs:
  - get
  - list
  - update
  - watch
  resourceNames:
  - openshift-console
```
but getting this error:
```
I1127 23:33:02.214834   42061 reflector.go:240] Listing and watching *v1.OAuthClientAuthorization from github.com/openshift/console-operator/vendor/github.com/openshift/client-go/oauth/informers/externalversions/factory.go:101
E1127 23:33:02.247762   42061 reflector.go:205] github.com/openshift/console-operator/vendor/github.com/openshift/client-go/oauth/informers/externalversions/factory.go:101: Failed to list *v1.OAuthClientAuthorization: oauthclientauthorizations.oauth.openshift.io "openshift-console" is forbidden: User "system:serviceaccount:openshift-console:console-operator" cannot list oauthclientauthorizations.oauth.openshift.io at the cluster scope: no RBAC policy matched
```
if i `oc login --token=<token.for.system:serviceaccount:openshift-console:console-operator>` i can `oc get oauthclient openshift-console`.   So its close, but not quite right.  I'm guessing then its something to do with `list` vs `get` but still fishing.